tests/Handler/StreamHandlerTest.php

```diff
@@ -554,6 +554,26 @@ public function testAddsContentLengthEvenWhenEmpty()
         self::assertEquals(0, $req->getHeaderLine('Content-Length'));
     }
 
+    public function testAddsContentLengthForPOSTEvenWhenEmpty()
+    {
+        $this->queueRes();
+        $handler = new StreamHandler();
+        $request = new Request('POST', Server::$url, [], '');
+        $handler($request, []);
+        $req = Server::received()[0];
+        self::assertEquals(0, $req->getHeaderLine('Content-Length'));
+    }
+
+    public function testDontAddContentLengthForGETEvenWhenEmpty()
+    {
+        $this->queueRes();
+        $handler = new StreamHandler();
+        $request = new Request('GET', Server::$url, [], '');
+        $handler($request, []);
+        $req = Server::received()[0];
+        self::assertSame('', $req->getHeaderLine('Content-Length'));
```

Comment:
This is the effect of my change, otherwise this would be
```suggestion
        self::assertSame(0, $req->getHeaderLine('Content-Length'));
```